src/effects/effectchainslot.h

```diff
@@ -133,11 +135,24 @@ class EffectChainSlot : public QObject {
     ControlPushButton* m_pControlChainEnabled;
     ControlObject* m_pControlChainMix;
     ControlObject* m_pControlChainSuperParameter;
+    ControlObject* m_pControlChainSuperForceUpdate;
     ControlPushButton* m_pControlChainInsertionType;
     ControlObject* m_pControlChainSelector;
     ControlPushButton* m_pControlChainNextPreset;
     ControlPushButton* m_pControlChainPrevPreset;
 
+    /**
```

Comment:
Can we look at fixing the problem that leads to this being the case instead? I think with the right set of fixes, ControllerEngine and the skin can always use the same control. 

I think we should change these two bits: 
https://github.com/mixxxdj/mixxx/blob/master/src/controllers/controllerengine.cpp#L532
https://github.com/mixxxdj/mixxx/blob/master/src/control/control.cpp#L128

Off the top of my head, here's a proposal that should work (I'm probably missing some corner cases):
- When a Control dies, it marks its ControlDoublePrivate as "abandoned".
- When ControllerEngine looks up a COScript, if the control is "abandoned" we delete the COScript and create a new one.

Optionally, abandoned COs could automatically clear their attached ControlProxy's with a signal so their dangling references get deleted, causing the CDP to get deleted. This would cause ControlProxy.valid() to return false, so ControlProxy could avoid dealing with "abandoned" at all. ControllerEngine could check "valid" every time it it looks up a COScript instead of the first time.